libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/view/MapboxManeuverView.kt

```diff
@@ -452,6 +456,15 @@ class MapboxManeuverView : ConstraintLayout {
                 )
             )
         )
+        upcomingManeuverAdapter.updateUpcomingManeuverIconStyle(
+            ContextThemeWrapper(
+                context,
+                typedArray.getResourceId(
+                    R.styleable.MapboxManeuverView_maneuverViewIconStyle,
```

Comment:
Let us add a new attribute in `attrs.xml` by the name `upcomingManeuverListIconStyle`.  This will allow end users to specify different colors for `maneuverViewIconStyle` and `upcomingManeuverListIconStyle`. Otherwise, if we use `maneuverViewIconStyle` for upcoming maneuvers in the list, changing the color for `maneuverViewIconStyle` will apply to all.